src/main/kotlin/no/nav/familie/ef/sak/mapper/OvergangsstønadMapper.kt

```diff
@@ -26,6 +27,15 @@ object OvergangsstønadMapper {
                                                dokumentasjon = tilDokumentasjonDto(situasjon.oppsigelseDokumentasjon))
     }
 
+    fun tilStønadsperiode(stønadsstart: Stønadsstart): StønadsperiodeDto {
+        val søkerStønadFra = if (stønadsstart.fraMåned != null && stønadsstart.fraÅr != null)
+            YearMonth.of(stønadsstart.fraÅr!!.verdi, stønadsstart.fraMåned!!.verdi) else null
+        return StønadsperiodeDto(
+                resterendeAvHovedperiode = "", // TODO
```

Comment:
Hva skal til for å få satt dette feltet? Er det informasjon som ikke finnes i søknaden?

Reply:
Confluence: Hovedperiode (36 mnd) minus tidligere brukt periode

Reply:
Da bør feltet settes til 36 mnd for førstegangssøkere? Så kan du heller skrive en TODO på at dette må beregnes for revurdering. Hvordan vil det se ut i frontend med et tomt felt der? Ser det bare står "X måneder" i zeplin, høres ut som noe som må avklares med design/fag :)

Reply:
Vi vet ikke akkurat nå om det er førstegangssøkere, så tenker vi kan la den være som en todo? Og også returnere "ikke impkementert" som tekst

Reply:
Ok, men skriv gjerne hva TODOen går ut på, hvis ikke er det umulig å forstå uten å gå inn i confluence... "Ikke implementert" er en bedre tekst :) Høres uansett ut som at vi må ta en diskusjon på hvordan vi avgjør om dette er en førstegangsbehandling eller revurdering når historikken ligger i Infotrygd. Hvis systemet ikke vil klare å avgjøre det, må vi ta en prat om hva vi skal gjøre med det feltet i frontend. Og selv om vi klarer å avgjøre det, vet vi at vi klarer å utlede riktig periode for revurderinger i versjon 0.1? Tone kan sikkert justere skissen etter hva hun mener er beste løsning hvis hun blir involvert her.

Reply:
99ad45b